lib/inspec/resources/docker_image.rb

```diff
@@ -48,6 +48,17 @@ def tag
       object_info.tags[0] if object_info.entries.size == 1
     end
 
+    def [](hashkeys)
```

Comment:
@ahasunos just think about the scenario its(['a.b', 'c.d' ]) on first look user will treat [] as array and not as method. Might need some better handling of this scenario.

Reply:
According to me, this whole thing is to be parsed as JSON and handle the keys like "a.b.c" at the backend while parsing data.

Reply:
In process to make the changes.

Reply:
@Vasu1105 Please have a look at the changes.